### PR TITLE
Correctly handle organizer images

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/96354

We do not show or use them... yet we should handle them correctly.

### CODE REVIEW

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 1]
```diff
@@ -2266,4 +2247,32 @@ public function should_queue_import( $should_queue_import = null ) {
...
+		$args = array(
+			'ID'         => $organizer_id,
+			'image'      => $service_image,
+			'post_title' => get_the_title( $organizer_id ),
+		);
+		$image = $this->import_image( $args );
+
+		if ( ! is_wp_error( $image ) && ! empty( $image->post_id ) ) {
+			// Set as featured image
+			$featured_status = $this->set_post_thumbnail( $organizer_id, $image->post_id );
```
Comment (barryhughes) [inline src/Tribe/Aggregator/Record/Abstract.php]: Could `$featured_image` or `$image_added` be a better name for this var?

**Comment by barryhughes** [comment]
Looks good, but we should add a changelog entry.

**Comment by lucatume** [comment]
Still working on this

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 2]
```diff
@@ -2266,4 +2247,47 @@ public function should_queue_import( $should_queue_import = null ) {
...
 	}
+
+	/**
+	 * Attaches a service-provided image to an organizer.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $organizer_id The organizer post ID.
+	 * @param string $service_image
+	 * @param \Tribe__Events__Aggregator__Record__Activity $activity
```
Comment (barryhughes) [inline src/Tribe/Aggregator/Record/Abstract.php]: No need to change it, but we don't really need a fully qualified class name here and could drop the leading `\`

**Pull request merged**